firmware/spade/src/rpi/upload.h

```diff
@@ -36,7 +70,7 @@ static struct {
 
 static void upl_flush_buf(void) {
   uint32_t interrupts = save_and_disable_interrupts();
-  flash_range_program(FLASH_TARGET_OFFSET + (upl_state.page++) * 256,
+  flash_range_program(FLASH_TARGET_OFFSET(slot) + (upl_state.page++) * 256,
```

Comment:
this seems to be modifying the engine code of the firmware

Reply:
oop, this was accidental